### DISCUSSION TITLE
关于删除插件 [自定义顶栏 - 频道] 的功能代码，仅保留 UI

### DISCUSSION THREAD

**Discussion by Tinhone** [the1812/Bilibili-Evolved#4623] [综合讨论 / General]
我看 #4614 提到了标题说的事情，那我可以水个 PR 嘛 ( ﾟ∀。)

**Comment by the1812** [comment]
我还是没明白仅保留 UI 的意义在哪里?

**Reply by Tinhone** [reply]
大概是赛博纪念碑吧

**Reply by the1812** [reply]
那可以去 Bilibili-Old 那水一个（
虽然那边的赛博纪念碑好久没更新了

**Reply by Geeyun-JY3** [reply]
可惜 [Bilibili-Old](https://github.com/MotooriKashin/Bilibili-Old) 在今年二月底归档了：
![](https://github.com/the1812/Bilibili-Evolved/assets/49396421/d9a6427e-962c-4775-82a2-b7c947b63d95)
作者的 status「🤣 庶务繁忙，有缘再会」或者已经说明了一切：
![](https://github.com/the1812/Bilibili-Evolved/assets/49396421/b21709d0-c593-4a6f-98f9-c87382283584)
